src/lib/sql/sql_pipeline_builder.cpp

```diff
@@ -40,7 +40,10 @@ SQLPipelineBuilder& SQLPipelineBuilder::disable_mvcc() { return with_mvcc(UseMvc
 SQLPipeline SQLPipelineBuilder::create_pipeline() const {
   DTRACE_PROBE1(HYRISE, CREATE_PIPELINE, reinterpret_cast<uintptr_t>(this));
   auto optimizer = _optimizer ? _optimizer : Optimizer::create_default_optimizer();
-  auto pipeline = SQLPipeline(_sql, _transaction_context, _use_mvcc, optimizer, _pqp_cache, _lqp_cache);
+  auto post_caching_optimizer =
+      _post_caching_optimizer ? _post_caching_optimizer : Optimizer::create_post_caching_optimizer();
```

Comment:
Consistency/Naming: `default_optimizer` vs `post_caching_optimizer` - what makes the first one "default"?